scripts/create-torrent

```diff
@@ -1,29 +1,35 @@
 #!/bin/sh
 
 set -e
+[ "$DEBUG" = "1" ] && set -x
```

Comment:
This will crash the script if DEBUG is `0` or empty. Either move before `set -e`, or convert to `if`.

Reply:
I tried, I don't have this problem

Reply:
It's also in the create-archive script